templates/ui-frameworks/svelte/web-app/ui/src/App.svelte.hbs

```diff
@@ -16,23 +17,25 @@ onMount(() => {
 
 <ClientProvider>
   <div>
-    <a href="https://developer.holochain.org/get-started/" target="_blank">
-      <img src={logo} class="logo holochain" alt="holochain logo" />
-    </a>
-  </div>
-  <h1>Holochain Svelte hApp</h1>
-  <div>
-    <div class="card">
-      {#if loading}
-      <p>connecting...</p>
-      {:else if error}
-      <p>{error.message}</p>
-      {:else}
-      <p>Client is connected.</p>
-      {/if}
+    <div>
```

Comment:
This looks like a big change but is just adding a wrapper div because the elements were shown side-by-side instead of below each other. Not actually related to the Svelte 5 bump but wanted to squeeze it in here for simplicity.